elftools/elf/relocation.py

```diff
@@ -322,6 +331,12 @@ def _reloc_calc_sym_plus_addend(value, sym_value, offset, addend=0):
     def _reloc_calc_sym_plus_addend_pcrel(value, sym_value, offset, addend=0):
         return sym_value + addend - offset
 
+    def _reloc_calc_value_plus_sym_addend(value, sym_value, offset, addend=0):
```

Comment:
If you rebase this PR, one of the recent commits made `_reloc_calc_sym_plus_value` do the same thing -- can that be used instead?

Reply:
Done, thanks!